Sources/App/Core/Search.swift

```diff
@@ -144,10 +144,11 @@ enum Search {
             packageName, coalesce(summary, emptyString), repoName, repoOwner, arrayToString(keywords, delimiter: " ")
         )
 
-        let exactPackageNameMatch = eq(lower(packageName), mergedTerms)
+        let exactPackageNameMatch = eq(lower(coalesce(packageName, emptyString)), mergedTerms)
         let sortOrder = SQLOrderBy(exactPackageNameMatch, .descending)
-            .then(tsrankvalue, .descending)
```

Comment:
Is moving `tsrankvalue` below the score intentional?

Reply:
Yes!

https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/2072#issuecomment-1277300588

Reply:
Sorry, I see this is also part of the discussion on the issue. Moving discussion there.